### PR TITLE
Update OctoPrint camera docs

### PR DESCRIPTION
**Pull request by slovdahl**
## Proposed change

Point users to the MJPEG IP Camera docs instead of having an example of the deprecated YAML config.

I set it up by following the docs and started getting this warning in the log:

```
2022-04-02 11:35:27 WARNING (MainThread) [homeassistant.components.mjpeg] Configuration of the MJPEG IP Camera platform in YAML is deprecated and will be removed in Home Assistant 2022.5; Your existing configuration has been imported into the UI automatically and can be safely removed from your configuration.yaml file
```

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/octoprint.markdown [hunk 1]
```diff
@@ -77,15 +77,7 @@ Supported sensors:
...
-If the OctoPrint host is equipped with a web camera it is possible to add this as well.
-
-```yaml
-camera:
-  - platform: mjpeg
-    name: OctoPrint
-    still_image_url: http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=snapshot
-    mjpeg_url: http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=stream
-```
+If the OctoPrint host is equipped with a web camera it is possible to add this as well using the [`MJPEG IP Camera`](/integrations/mjpeg) integration. Use `http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=stream` for the MJPEG URL and `http://YOUR_OCTOPRINT_HOST_IP/webcam/?action=snapshot` as the still image URL.
```
Comment (slovdahl) [inline source/_integrations/octoprint.markdown:80]: Is it possible (or desirable) to add a link for adding the integration directly? 🤔 Similar to the blue "ADD INTEGRATION" button 👇 but for the MJPEG IP Camera integration.

image

Comment (slovdahl) [reply source/_integrations/octoprint.markdown:80]: Reverse engineering the include gave me this:

```
{% my config_flow_start badge domain=page.ha_domain %}
```

So I guess something like

```
{% my config_flow_start badge domain=mjpeg %}
```

could work. Then the question is only if it's the right thing to do 🙂

Comment (slovdahl) [reply source/_integrations/octoprint.markdown:80]: Added it now to test if it works, but I can revert it if it's considered more confusing/harmful than anything else.

**Review by frenck — APPROVED** [review]
Thanks, @slovdahl 👍

**Pull request merged**